test/test_ops.py

```diff
@@ -1702,6 +1705,16 @@ def test_refs_are_in_python_ref_db(self, op):
             return
         self.assertIn(op, self.ref_db_names_set)
 
+    # like 'test_refs_are_in_python_ref_db' but check in the other direction,
+    # this includes aliases since those are not allowed in 'ref_db_names'
+    @parametrize("op", itertools.chain(ref_db_names, ref_alias_names))
+    def test_python_refs_are_in_ops_names(self, op):
+        # TODO: skip nvfuser refs since those don't have __all__ and it's not
+        # clear at the moment what would be the best way to check those
+        if op.startswith("ops.nvprims"):
+            return
```

Comment:
Could you please fill an issue and reference it here?

Reply:
https://github.com/pytorch/pytorch/issues/85081